libs/librrgraph/src/io/gen/rr_graph_uxsdcxx.h

```diff
@@ -1625,7 +1625,7 @@ template<std::size_t N>
 constexpr const char *lookup_switch_type[] = {"UXSD_INVALID", "mux", "tristate", "pass_gate", "short", "buffer"};
 constexpr const char *lookup_segment_res_type[] = {"UXSD_INVALID", "GENERAL", "GCLK"};
 constexpr const char *lookup_pin_type[] = {"UXSD_INVALID", "OPEN", "OUTPUT", "INPUT"};
-constexpr const char *lookup_node_type[] = {"UXSD_INVALID", "CHANX", "CHANY", "SOURCE", "SINK", "OPIN", "IPIN"};
+constexpr const char *lookup_node_type[] = {"UXSD_INVALID", "CHANX", "CHANY", "SOURCE", "SINK", "OPIN", "IPIN", "MEDIUM"};
```

Comment:
These are auto-generated files. Ensure you have modified the source files.  
https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/39c80f444fb37b81681294aadae6dc518f79ac8b/libs/librrgraph/src/io/rr_graph.xsd#L228-L237